contents/careers/marketer.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Marketer
+sidebar: Careers
+showTitle: true
+---
+## PostHog exists to increase the number of successful products in the world.
+
+We're growing quickly and are looking for our first marketer to…… While we have mainly engineers to date, we are now hiring across all areas of the business.
+We also would ideally love to bring on someone who is interested in xxxx.
+
+## What you’ll be doing
+*
```

Comment:
Things that are important here off top of head:

* Ability to understand a developer's mindset. Being an ex-developer I'd consider very important given our stage. The only alternative would be for them to collaborate very closely with our engineering team but we've a preference for more "full stack" type people. 
* Marketing experience (not a newbie, this needs to be someone that already knows what to do)
* Digital-oriented (we'll end up in addition hiring a PR/credibility/brand type marketer next - who will be more PR/events/pbulicity stunt focussed)
  * Paid experience is a plus
  * Social experience is important
  * Content experience is important (this is why they will need to be a developer to really nail this as our audience will need decent technical content or content that would do well on websites like Hacker News)
  * SEO won't help - this will come from doing a good job of PR later, so isn't a current priority
* Profile wise:
  * Important they've worked at early stage companies and have been there as they've grown. Maybe ex-founder or very early stage employee.
  * Experience at developer-focussed companies in marketing also a big plus (this could reduce need for them to be more technical)
  * If they want to move to be a manager explicitly, not sure it'll be a great fit long term but it's not a complete no